examples/css-minifier/webpack.config.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+
+module.exports = {
+  mode: 'production',
+  entry: ['./src/style.css'],
+  optimization: {
+    minimizer: [new OptimizeCSSAssetsPlugin({
+      assetNameRegExp: /\.css$/,
+      cssProcessor: require('cssnano'),
+      cssProcessorPluginOptions: {
+        preset: ['default', { discardComments: { removeAll: true } }],
+      },
+      canPrint: true
+    })],
+  },
+  plugins: [
+    new MiniCssExtractPlugin({
+      filename: '[name].[hash].css',
+      chunkFilename: '[id].[hash].css',
+    }),
+  ],
```

Comment:
Try using the `webpack-file-preprocessor-plugin` here whose use case was to be denostrated. Please consider checking this [example](https://github.com/kabirbaidhya/webpack-file-preprocessor-plugin/blob/master/examples/html-minifier/webpack.config.js).

Reply:
sorry i misunderstood the task. Will update the PR.